appveyor.yml

```diff
@@ -20,7 +20,7 @@ install:
     - vcpkg upgrade --no-dry-run
     - vcpkg list
     - vcpkg install
-          boost-algorithm boost-accumulators boost-atomic boost-container boost-date-time boost-exception boost-filesystem boost-graph boost-log boost-program-options boost-property-tree boost-ptr-container boost-regex boost-serialization boost-system boost-test boost-thread
+          boost boost-algorithm boost-accumulators boost-atomic boost-container boost-date-time boost-exception boost-filesystem boost-graph boost-log boost-program-options boost-property-tree boost-ptr-container boost-regex boost-serialization boost-system boost-test boost-thread
```

Comment:
Is this obsoleted by https://github.com/alicevision/AliceVision/pull/842/commits/7252e4234c38290661171db0afa590f59470fc0b ?

Reply:
No, the "boost" target in vcpkg means all boost modules, so we should not include it.

See the Build-Depends here: https://github.com/microsoft/vcpkg/blob/master/ports/boost/CONTROL#L7

Reply:
What you were probably trying to do is adding boost-core:
https://github.com/microsoft/vcpkg/blob/master/ports/boost-core/CONTROL#L7

But it's already a dependency of the other modules. For instance: here https://github.com/microsoft/vcpkg/blob/master/ports/boost-array/CONTROL#L4

Reply:
Hmm, I think it may be best to drop this patch and submit it separately. I added it as CI was failing on Windows, but it's really unrelated. I'll split this out into a separate PR.